.travis.yml

```diff
@@ -1,14 +1,11 @@
 language: java
+
+dist: xenial
+
 jdk:
-  - oraclejdk9
-
-matrix:
-  include:
-    - jdk: openjdk9
-      before_install:
-        - rm "${JAVA_HOME}/lib/security/cacerts"
-        - ln -s /etc/ssl/certs/java/cacerts "${JAVA_HOME}/lib/security/cacerts"
-        
+- openjdk8
```

Comment:
Big thanks for fixing the Travis build! 
I do want to move to java 9+ but having it green now is definitely more important.

Reply:
It's not about moving to java9, it's about getting rid of cobertura, because that tool ain't supporitng JDK > 8. Maybe JaCoCo ?